madara/crates/primitives/rpc/src/v0_7_1/starknet_api_openrpc.rs

```diff
@@ -588,9 +588,9 @@ pub struct ExecutionResources {
 #[derive(Clone, Debug, Eq, Hash, PartialEq, Serialize, Deserialize)]
 pub struct DataAvailability {
```

Comment:
what was this thinggy again? why was it named DA?
(also: wtf, we were returning u128s in jsonrpc? json numbers dont go that far right?)